<launch>
  <node pkg="mission_pkg" name="mission" type="bt_ros" output="screen" >
    <param name="config_path" value="$(find mission_pkg)/config/land.xml" type="str"/>
    <param name="fcu_state_topic" value="/mavros/state" type="str"/>
    <param name="fcu_pos_topic" value="/mavros/local_position/local" type="str"/>
    <param name="pos_tgt_topic" value="/mavros/setpoint_raw/local" type="str"/>
    <param name="set_mode_service" value="/mavros/set_mode" type="str"/>
    <param name="arm_service" value="/mavros/cmd/arming" type="str"/>
  </node>
</launch>
